data/index.js

```diff
@@ -104,5 +119,5 @@ export const query = ( mapSelectorsToProps ) => ( WrappedComponent ) => {
  * @returns {*} The selector's returned value.
  */
 export const select = ( reducerKey, selectorName, ...args ) => {
-	return selectors[ reducerKey ][ selectorName ]( store.getState()[ reducerKey ], ...args );
+	return selectors[ reducerKey ][ selectorName ]( stores[ reducerKey ].getState(), ...args );
```

Comment:
In line 103 we have the following:

```js
const select = ( key, selectorName, ...args ) => {
	return selectors[ key ][ selectorName ]( state[ key ], ...args );
};
```
I was wondering if we could extract shared function, but I have no clue how to do it in a way that will simplify code instead of making it more complex 😛

Reply:
I agree that some abstraction is needed here. Also, I think this function needs some safeguards to preventing exceptions when non-existing selectors/stores are called.